app-server/nginx-with-rr.md

```diff
@@ -157,21 +157,18 @@ In this example, we will demonstrate how to use RoadRunner with Nginx in a Docke
 
 ```docker
 FROM --platform=${TARGETPLATFORM:-linux/amd64} ghcr.io/roadrunner-server/roadrunner:latest as roadrunner
```

Comment:
You may add a warning here about the latest tag with:
{% hint style="warning" %}
Consider using the direct version in production. `latest` image tag should be used in the development environments only. (something like this).
{% endhint %}

Reply:
No, that's wrong. Development environments should be the same as production.